modules/json/src/json-loader.js

```diff
@@ -0,0 +1,141 @@
+/* global __VERSION__ */ // __VERSION__ is injected by babel-plugin-version-inline
+/* global TextDecoder */
+import {TableBatchBuilder, RowTableBatch} from '@loaders.gl/tables';
+import StreamingJSONParser from './lib/streaming-json-parser';
+
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+export const JSONLoader = {
+  id: 'json',
+  name: 'JSON',
+  version: VERSION,
+  extensions: ['json'],
+  mimeType: 'text/json',
+  /*
+  extensions: {
+    json: null,
+    jsonl: {stream: true},
+    ndjson: {stream: true}
+  },
+  mimeTypes: {
+    'application/json': null,
+    'application/json-seq': {stream: true},
+    'application/x-ndjson': {stream: true}
+  },
+  */
+  category: 'table',
+  parse: async (arrayBuffer, options) =>
```

Comment:
Why is a function that calls `parseJSONSync` async?

Reply:
The convention is that all loaders have at least an async `parse` function. 

But yes, I suppose we should also offer a `parseSync`.